### PR TITLE
Remove useless third party and clean module code

### PR DESCRIPTION
**Pull request by Matt75**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Clean code, remove useless components, decrease ajax calls, reuse core notification controller, reuse core components, setup devtools
| Type?         | refacto
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | Everything should work as before

Sadly I can't use the same ajax call made by core to update notifications counters because no event is fire... So I have to keep module ajax call, it's a duplicate but no better solution today.
This module is only compliant with 1.7.6+ since beginning, no change.

### CODE REVIEW

**Comment by Matt75** [comment]
PHP-CS-Fixer and PHPStan are now happy 🥳

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+FILEPATH=$1
+ENV=$2
+SHOP=$3
+
+gcloud container clusters get-credentials --project="prestashop-ready-$ENV" --zone="europe-west1-d" "prestashop-ready-cluster"
```
Comment (PierreRambaud) [inline deploy.sh:7]: This file need to be remove, it's useless :thinking:

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: '3.7'
+
+services:
+  php:
+    image: phpdockerio/php72-cli
+    volumes:
```
Comment (PierreRambaud) [inline docker-compose.yml:6]: Is this file needed?

File: controllers/admin/AdminConfigureFaviconBoController.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        $this->table = 'configuration';
+
+        parent::__construct();
+
+        $this->fields_options = [
+            'ps_faviconnotificationbo' => [
+                'title' => $this->trans('Configuration', [], 'Admin.Global'),
+                'info' => $this->trans('Display notifications in the browser tab for:', [], 'Modules.Faviconnotificationbo.Admin'),
+                'fields' => [
+                    'CHECKBOX_ORDER' => [
```
Comment (PierreRambaud) [inline controllers/admin/AdminConfigureFaviconBoController.php:35]:
```suggestion
                    static::CHECKBOX_ORDER => [
```

Should be a const, because also used in ps_faviconnotificationbo.php, same for the others

File: upgrade/upgrade-2.1.0.php [hunk 4]
```diff
@@ -27,7 +26,7 @@
  *
  * @return bool
  */
-function upgrade_module_2_0_1($module)
+function upgrade_module_2_1_0($module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.1.0.php:29]: Are you sure?

Comment (Matt75) [reply upgrade/upgrade-2.1.0.php:29]: Yes, this upgrade file has been added yesterday and version 2.0.1 never released

File: deploy.sh (hunk 1 above)
Comment (Matt75) [reply deploy.sh:7]: Oops yes

File: controllers/admin/AdminConfigureFaviconBoController.php (hunk 3 above)
Comment (Matt75) [reply controllers/admin/AdminConfigureFaviconBoController.php:35]: Yes I will add const instead

File: docker-compose.yml (hunk 2 above)
Comment (Matt75) [reply docker-compose.yml:6]: Checked with others devs, not needed

File: upgrade/upgrade-2.1.0.php (hunk 4 above)
Comment (PierreRambaud) [reply upgrade/upgrade-2.1.0.php:29]: ohh I understand :)

**Comment by sarahdib** [comment]
Hello @Matt75 

There is an error with color picker 
Capture d’écran 2020-06-05 à 17 52 42
Capture d’écran 2020-06-05 à 17 53 14

If I disable only one option the favicon notification is disable for all 
Capture d’écran 2020-06-05 à 18 00 21

**Comment by Progi1984** [comment]
@Matt75 Could you fix the conflicting file ? After that, we merge your PR.
